common/djangoapps/util/json_request.py

```diff
@@ -66,7 +66,7 @@ def __init__(self, resp_obj=None, status=None, encoder=EDXJSONEncoder,
         elif isinstance(resp_obj, QuerySet):
             content = serialize('json', resp_obj)
         else:
-            content = json.dumps(resp_obj, cls=encoder, indent=2, ensure_ascii=False)
+            content = json.dumps(resp_obj, cls=encoder, indent=2, ensure_ascii=True)
```

Comment:
@mushtaqak This seems to fix the issues you, @robrap, and ! were discussing on your PR earlier: https://github.com/edx/edx-platform/pull/12237/files#r63996545. We're going to have a full test run and see if this is an acceptable default change to make before evaluating our options further.

@nedbat Is there a reason that `ensure_ascii` would need to be False here? It looks like we do this in several places: https://github.com/edx/edx-platform/search?&q=ensure_ascii, but the `dump_js_escaped_json` method uses True already https://github.com/edx/edx-platform/blob/master/openedx/core/djangolib/js_utils.py#L72